scalyr_agent/scalyr_client.py

```diff
@@ -151,6 +151,7 @@ def create_client(config, quiet=False, api_key=None):
         disable_send_requests=config.disable_send_requests,
         disable_logfile_addevents_format=config.disable_logfile_addevents_format,
         enforce_monotonic_timestamps=config.enforce_monotonic_timestamps,
+        workers_api_keys_tuple=config.get_number_of_configured_workers_and_api_keys(),
```

Comment:
Technically I could also pass in the whole configuration object here, but to follow the existing API and be on the safe-side with potential cyclic dependencies, I went with this approach.

I think cyclic dependency should not be an issue even if I passed in the whole object since config + client gets re-created when config change is detected, but just in case.

Reply:
That sounds like the right call to me.